### DISCUSSION TITLE
GitHub Sponsor not through an external website?

### DISCUSSION THREAD

**Discussion by dorianmariecom** [rubygems/rubygems.org#2863] [Ideas]
I would rather donate if it was managed by GitHub

**Comment by sonalkr132** [comment]
you should ask this in slack.

**Reply by dorianmariecom** [reply]
done

**Comment by indirect** [comment]
You can support RubyGems via GitHub Sponsors at https://github.com/sponsors/rubytogether.

**Reply by dorianmariecom** [reply]
thanks
